universe/Effect.h

```diff
@@ -907,11 +907,20 @@ class FO_COMMON_API Effect::GenerateSitRepMessage : public Effect::EffectBase {
     GenerateSitRepMessage(const std::string& message_string, const std::string& icon,
                           const std::vector<std::pair<std::string, ValueRef::ValueRefBase<std::string>*> >& message_parameters,
                           ValueRef::ValueRefBase<int>* recipient_empire_id,
-                          EmpireAffiliationType affiliation);
+                          EmpireAffiliationType affiliation,
+                          const std::string label = "",
+                          bool no_stringtable_lookup = false);
```

Comment:
The preexisting parser token was "NoStringtableLookup", and the preexisting use for ShipDesigns, which you wrote less than a year ago, had this exact same polarity, at least at the initial parser level. But you did reverse the polarity before passing it on to ShipDesigns, true.  However, the code for handling the polarity reversal is a bit simpler if you wait until slightly later like I did.  But if you need me to reverse the polarity earlier like you did, ok, I'll change it. Seems rather arbitrary though.